src/com/projectkorra/projectkorra/Element.java

```diff
@@ -139,7 +140,7 @@ public ElementType getType() {
 
 	@Override
 	public String toString() {
-		return this.getColor() + this.getName();
+		return this == Element.BLUE_FIRE ?  this.getColor() + "Blue Fire": this.getColor() + this.getName();

```

Comment:
Should be written like this: `(this == Element.BLUE_FIRE) ? this.getColor() + "Blue Fire" : this.getColor() + this.getName();`

Reply:
I think a more general case should be made where underscores are replaced with spaces, rather than just a specific exception for Blue Fire

Reply:
I agree with you, but we aren't going to be adding anything else to this.